learn/tanssi/technical.md

```diff
@@ -3,51 +3,83 @@ title: Technical Features
 
-The Polkadot Relay Chain serves as the main network that coordinates the overall operation of the platform providing shared security, consensus, and interoperability amongst the connected parachains, to accelerate the development of the web3.
+![Sibling Chains](/images/learn/tanssi/technical/light-technical-2.png#only-light)
+![Sibling Chains](/images/learn/tanssi/technical/dark-technical-2.png#only-dark)
 
-## The technology behind
+### Relay Chain {: #relay-chain } 
 
-Polkadot -and the parachains- are built with an open-source blockchain development framework called Substrate, which is bases on Rust programming language.
+Among many other responsibilities, the relay chain validates and finalizes the blocks produced by any chain participating in the ecosystem (including the ContainerChains and Tanssi Network), storing the most recent headers for each block of each chain.
 
-Substrate makes it easy for teams to develop blockchains, providing pre built core functinalities such as consensus mechanisms, staking & governance, account and assets management, networking and many more ready to use modules and libraries.
+Tanssi Protocol leverages this feature, relying on the relay chain as a means to provide the necessary data to both, Tanssi and its ContainerChains, allowing them to collaborate and validate the correctness of the block production service.
 
-The core principles of Substrate are ease of use and performance, allowing teams to create specialized blockchains for any use case, with high level of performance, flexibility, and robustness.
\ No newline at end of file
+### Tanssi Network {: #tanssi-newtwork } 
+
+As previously discussed, Tanssi Protocol assigns collators to Tanssi Network itself and the ContainerChains, and the result of this assignment is stored within the chain state.
+
+Another important piece of information that Tanssi stores is the latest header for every ContainerChain. This data is read from the relay chain and, being stored in every Tanssi block, it allows the protocol to keep track of the state in every chain and also to identify and reward accordingly the collator that produced their last block.
+
+### ContainerChain {: #containerchain } 
+
+As a collator node assigned to a ContainerChain deployed in Tanssi, and having a built-in Tanssi node functionality, it is technically feasible to read the state from the relay chain and the state from the Tanssi Network.
+
+Leveraging this ability to access the states, the current collator with the authority to produce a block will read the latest block produced in the relay chain which contains the state root of the latest block produced in Tanssi. With this state root, it will proceed to read the state in Tanssi, and include in the block of the ContainerChain the latest state root of Tanssi Network, the current set of collators assigned to the ContainerChain, and its public signature, allowing Tanssi to know who produced the block and reward the collator.
```

Comment:
```suggestion
Leveraging this ability to access the states, the current collator with the authority to produce a block will read the latest block produced in the relay chain, which contains the state root of the latest block produced in Tanssi. With this state root, it will proceed to read the state in Tanssi, and include in the block of the ContainerChain the latest state root of Tanssi Network, the current set of collators assigned to the ContainerChain, and its public signature, allowing Tanssi to know who produced the block and reward the collator.
```